template/package.json

```diff
@@ -11,25 +11,28 @@
     "watch": "zimlet watch",
     "start": "zimlet watch",
     "package": "zimlet package",
-    "test": "eslint src",
+    "lint": "eslint src",
```

Comment:
we need to update this in circleci config

Reply:
https://github.com/Zimbra/zm-x-zimlet-sideloader/pull/2/files#diff-1d37e48f9ceff6d8030570cd36286a61R48